<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>632</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>632</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>700</width>
    <height>632</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hasta Ekle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 152);
</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>266</width>
     <height>669</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/hospital.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/logs/4767662091595341176-128.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Hasta Soyadı :</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="soyad_Line">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhLowercaseOnly</set>
      </property>
      <property name="placeholderText">
       <string>Soyad</string>
      </property>
     </widget>
    </item>
    <item row="30" column="0" colspan="2">
     <widget class="QPushButton" name="hastaEkle_Btn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 190);</string>
      </property>
      <property name="text">
       <string>Onayla</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Doğum Tarihi (Yıl):</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="mail_Line">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhLowercaseOnly</set>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QComboBox" name="comboBox_Cinsiyet">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <item>
       <property name="text">
        <string>Kadin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Erkek</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cinsiyet</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QDateEdit" name="rt_dateEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="minimumDate">
       <date>
        <year>2021</year>
        <month>6</month>
        <day>2</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2021</year>
        <month>6</month>
        <day>4</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Randevu Tarihi :</string>
      </property>
     </widget>
    </item>
    <item row="22" column="1">
     <widget class="QComboBox" name="comboBox_Saat">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Randevu Saati :</string>
      </property>
     </widget>
    </item>
    <item row="23" column="1">
     <widget class="QComboBox" name="comboBox_Poliklinik">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Poliklinik :</string>
      </property>
     </widget>
    </item>
    <item row="24" column="1">
     <widget class="QComboBox" name="comboBox_Doktor">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
    </item>
    <item row="24" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Doktor :</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Hasta Adı :</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="ad_Line">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Ad</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_8">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TC Kimlik Numarası :</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="tc_Line">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>TC</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QDateEdit" name="dt_dateEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>12</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Icons/hasta.qrc"/>
 </resources>
 <connections/>
</ui>
